src/installer/tests/Assets/Projects/HostApiInvokerApp/Program.cs

```diff
@@ -30,6 +30,10 @@ public static void MainCore(string[] args)
         {
             Console.WriteLine("Hello World!");
             Console.WriteLine(string.Join(Environment.NewLine, args));
+            if (args.Length == 0)
+            {
+                throw new Exception("Invalid number of arguments passed");
+            }
```

Comment:
Is the above output appropriate given we are about to fail?

Reply:
> Do you mean to move the args printing to after this check?

Yep. It seems odd to use the args but then validate them and throw an exception.